pkg/boot/initrd.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package boot
+
+import (
+	"bytes"
+	"io"
+
+	"github.com/u-root/u-root/pkg/uio"
+)
+
+// PadInitrds pads initrds from a list of io.Readers to a 512 byte boundary
+// and returns a uio.LazyOpenerAt
+func PadInitrds(initrds []io.Reader) *uio.LazyOpenerAt {
```

Comment:
oh, one more. let's call it `CatInitrd` rather than Pad for concatenating? padding just happens to be how that is implemented